js/deer-config.js

```diff
@@ -156,18 +156,25 @@ export default {
                 <p class="filterNotice is-hidden"> Gloss filter detected.  Please note that Glosses will appear as they are fully loaded. </p>
                 <div class="totalsProgress" count="0"> {loaded} out of {total} loaded (0%).  This may take a few minutes.  You may click to select any Gloss loaded already.</div>
             </div>`
-            
+            function removeDuplicates(list, uniqueProp) {
+                return list.filter((item, index, self) =>
+                    index === self.findIndex((t) => (
+                        t[uniqueProp] === item[uniqueProp]
+                    ))
+                );
+            }
             // Grab the cached expanded entities from localStorage.  Note that there is nothing to check on "staleness"
             const cachedFilterableEntities = localStorage.getItem("expandedEntities") ? new Map(Object.entries(JSON.parse(localStorage.getItem("expandedEntities")))) : new Map()
+            const deduplicatedList = removeDuplicates(obj[options.list], '@id')
```

Comment:
You should be doing this inside `if(options.list)`.  If there is no `options.list` here, which is possible, you will get a null pointer exception.